ui-core/src/main/res/values/styles.xml

```diff
@@ -22,42 +25,44 @@
         <item name="android:textColorPrimary">?attr/colorOnBackground</item>
         <item name="android:textColorSecondary">?attr/colorOnBackground</item>
         <item name="android:textColorTertiary">?attr/colorOnBackground</item>
+
+        <item name="bottomSheetDialogTheme">@style/Adyen.Checkout.BottomSheetDialogTheme</item>
     </style>
 
     <!-- Base theme for Checkout -->
-    <style name="AdyenCheckout" parent="AppTheme" />
+    <style name="Adyen.Checkout" parent="Adyen.AppTheme" />
```

Comment:
Is there a benefit from renaming `AdyenCheckout` to `Adyen.Checkout`? I'm worried it will add more migration steps for merchants and (correct me if i'm wrong) if they were already overriding our styles they will not get a build error when they update so it won't be obvious that they need to rename `AdyenCheckout` to `Adyen.Checkout` 🤔

Reply:
Yeah I agree with you, let me revert it. I was experimenting with theme inheritance, but in the end what I wanted wasn't possible. Thought it looked kinda nice, so left it in